builtin/rebase.c

```diff
@@ -1404,9 +1415,9 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 		OPT_PASSTHRU_ARGV(0, "ignore-whitespace", &options.git_am_opts,
 				  NULL, N_("passed to 'git am'"),
 				  PARSE_OPT_NOARG),
-		OPT_PASSTHRU_ARGV(0, "committer-date-is-author-date",
-				  &options.git_am_opts, NULL,
-				  N_("passed to 'git am'"), PARSE_OPT_NOARG),
+		OPT_BOOL(0, "committer-date-is-author-date",
+			 &options.committer_date_is_author_date,
+			 N_("lie about committer date")),
```

Comment:
Sure, but there are lots of ways to lie about the committer date; this isn't the only one.  The description needs a bit more precision, e.g. "make committer date match author date".